scheduler/src/cook/mesos/schema.clj

```diff
@@ -558,6 +591,11 @@
     :db/ident :instance/create
     :db/doc "Creates an instance for a job"}])
 
+(def straggler-handling-types 
+  (->> schema-attributes
+       (map :db/ident)
+       (filter #(= "straggler-handling.type" (namespace %)))))
+
```

Comment:
I think we should switch hp-types to match straggler-handling. The ident as enum is a pretty standard pattern in datomic.